DIE/die.html

```diff
@@ -0,0 +1,4734 @@
+								<input type="checkbox" name="attr_Ecstacy" style="grid-row: 2/span 1;grid-column: 4/span 1;">
+								<input type="checkbox" name="attr_Admiration" style="grid-row: 3/span 1;grid-column: 5/span 1;">
+								<input type="checkbox" name="attr_Terror" style="grid-row: 4/span 1;grid-column: 6/span 1;">
+								<input type="checkbox" name="attr_Amazement" style="grid-row: 5/span 1;grid-column: 5/span 1;">
+								<input type="checkbox" name="attr_Grief" style="grid-row: 6/span 1;grid-column: 4/span 1;">
+								<input type="checkbox" name="attr_Loathing" style="grid-row: 5/span 1;grid-column: 3/span 1;">
+								<input type="checkbox" name="attr_Rage" style="grid-row: 4/span 1;grid-column: 2/span 1;">
+								<input type="checkbox" name="attr_Vigilance" style="grid-row: 3/span 1;grid-column: 3/span 1;">
+							</div>
+						</div>
+					</div>
+					<div class="sheet-2colrow">
+						<div class="sheet-col">
+							<h3>The Performance</h3>
+							<p>
+								To use The Voice, pick the emotion(s) and roll a <b>Cha</b> dice pool plus your D4. Lose 1 success for each target past the first. The D4 shows the effect’s <b>Intensity</b>, and each successes lets you add or subtract 1 from it. The final <b>Intensity</b> affects the <b>Audience</b> according to your goal – Control or Attack.
+							</p>
+							<p>
+								<b>Control:</b> Manipulate them with the emotion and steal their agency. +1 Disadvantage per <b>Intensity</b> on actions opposing the compulsion, +1 Advantage on actions that support it. Complete control if <b>Intensity</b> exceeds <b>Willpower</b>.
+								<br>
+								<b>Attack:</b> Make them feel so hard it hurts. They take +1 Disadvantage per <b>Intensity</b> on attacks, and are killed or incapacitated if <b>Intensity</b> exceeds <b>Willpower</b>.
+							</p>
+							<p>
+								Pass your D4 to a player whose character you’re influencing. To affect anyone else with your powers, you must reclaim your D4. This is instant and ends that compulsion.
+							</p>
+						</div>
+						<div class="sheet-col">
+							<h3>INTENSITY OF EFFECT</h3>
+							<table>
+								<tr>
+									<td style="width: 10%"><b>1</b></td>
+									<td>Enough to influence them.</td>
+								</tr>
+								<tr>
+									<td style="width: 10%"><b>2</b></td>
+									<td>They feel it intensely.</td>
+								</tr>
+								<tr>
+									<td style="width: 10%"><b>3</b></td>
+									<td>They feel it as intensely as most people will ever feel it.</td>
+								</tr>
+								<tr>
+									<td style="width: 10%"><b>4</b></td>
+									<td>More intense than love for a soulmate.</td>
+								</tr>
+								<tr>
+									<td style="width: 10%"><b>5</b></td>
+									<td>All-consuming obsession.</td>
+								</tr>
+								<tr>
+									<td style="width: 10%"><b>6+</b></td>
+									<td>Supernaturally intense. Likely lethal.</td>
+								</tr>
+							<table>
```

Comment:
We saw you were using tables, and I noticed this one doesn't have a proper closing tag while running `ctrl-f` to ensure you weren't [breaking our policy](https://wiki.roll20.net/Designing_Character_Sheet_Layout#HTML_Table)!
```suggestion
							</table>
```